### ISSUE TITLE
Logix Node Request: Ignore Right Controller/Left Controller

### ISSUE THREAD

**Issue by GearBell** [Frooxius/NeosPublic#632]
There has been a few unique situations where there is a grabbable item hooked to the hand, and is accidentally being grabbed constantly, same for an interactive button. If we had a new node to logix-wise make a [summarized example for the grabbable situation] GetActiveUser-GetControllers-IgnoreRight/Left Controller-Grabbable, it would solve a few things. That way the item on the left hand can be ignored by the left hand, but still interactive with the right hand, etc. This would be useful for grabbables and interactives.

**Comment by H3BO3** [comment]
I feel like this would be better as component options on interactables. Logix nodes make no sense as the grabbing isn't done through them

**Comment by GearBell** [comment]
for the situations i need, it cant be a component. it must be logix driven because its reliant on conditions. If (condition true) then (ignore controller), If (condition false) then (do not ignore controller). (Condition activate)-delay 2 seconds - Ignore Controller. Etc. A node would be more controlling on conditions.

**Comment by H3BO3** [comment]
You can drive/write component properties based on conditions.

**Comment by Hayden-da-Serg** [comment]
> for the situations i need, it cant be a component.
For that situation, the properties in the Grabbable component would be exactly what you need, and that was what my original idea was, was to have an Ignore Left and Ignore Right value in the Grabbable components.

**Comment by H3BO3** [comment]
Exactly. Logix nodes aren't necessary, there could just be an "IgnoreChirality" setting on the interactables and you could set it to left/right/none or similar, manually or through logix

**Comment by Frooxius** [comment]
Hmm I'm not fully sure how would the LogiX node be involved in this. The grabbing itself isn't handled through LogiX. I do want to add a mechanism to hook up custom grab check via LogiX, but that's pretty big general addition in the future, so that will take a while.

I could add flags to the Grabbable and other components that would let you disable interactions with particular hand if that would help specifically for that functionality.

**Comment by GearBell** [comment]
as long as we can logix drive those flags, sure

**Comment by H3BO3** [comment]
Bumping this as it seems it wasn't addressed yet, it's been open for a while now and seems like a fairly simple change

**Comment by DeliriousJax** [comment]
I constantly accidentally grab my left arm gadgets with my left hand. Whatever a solution is found, it'd be nice to have this figured out ;~;

**Comment by GearBell** [comment]
leaving another poke here to refresh the issue. needed badly for unusual rigs and gadgets.
